chimney/src/main/scala/io/scalaland/chimney/internal/TransformerDerivationError.scala

```diff
@@ -47,6 +48,12 @@ final case class IncompatibleSourceTuple(
 final case class NotSupportedTransformerDerivation(fieldName: String, sourceTypeName: String, targetTypeName: String)
     extends TransformerDerivationError
 
+final case class MacroException(exception: Throwable, sourceTypeName: String, targetTypeName: String)
```

Comment:
`MacroException`/`NotYetImplemented` don't seem to be transformer-specific errors. How about extracting them to a separate hierarchy?

Reply:
duplicate of https://github.com/scalalandio/chimney/pull/294#discussion_r1170509241